src/IECoreHoudini/plugin/Plugin.cpp

```diff
@@ -98,36 +98,60 @@ void newSopOperator(OP_OperatorTable *table)
 {
 	OP_Operator *opHolder = new OP_Operator(
 		"ieOpHolder", "Cortex Op",
-		SOP_OpHolder::create, SOP_ParameterisedHolder::parameters, 0, 4,
-		SOP_ParameterisedHolder::variables, OP_FLAG_GENERATOR
+		SOP_OpHolder::create,
+		SOP_ParameterisedHolder::parameters,
+#if UT_MAJOR_VERSION_INT >= 16
+		SOP_OpHolder::theChildTableName,  
+#endif
+		0, 4, 0, 
+		OP_FLAG_GENERATOR | OP_FLAG_NETWORK
```

Comment:
OP_FLAG_GENERATOR tells houdini that this node creates geometry. I'll take it out for the nodes that aren't creating geometry though.